BUILD.bazel

```diff
@@ -194,18 +213,33 @@ ogre_gl_support_srcs = glob(
 )
 
 cc_library(
-   name = "rendersystem_gl",
-   srcs = ogre_gl_support_srcs,
-   hdrs = ogre_gl_support_hdrs,
+   name = "rendersystem_gl3plus",
+   srcs = ogre_gl_support_srcs + ogre_gl_support_hdrs,
+   hdrs = ogre_gl_support_hdrs + ["OgreMain/include/OgreBuildSettings.h"],
    includes = ogre_gl_support_includes,
+   deps = [ ":ogre_main_headers" ],
    copts = [
        "-Wno-implicit-fallthrough",
    ],
-   deps = [
-       ":ogre_main",
-   ],
-   alwayslink = 1,
+   linkopts = [
+       "-lGL", "-lGLU",
+       "-lSM", "-lICE",
+       "-lX11", "-lXext",
+       "-lXrandr", "-lXt",
+       "-lXaw", "-lpthread",
+       "-ldl", "-lfreeimage", "-latomic"],
    visibility = ["//visibility:public"],
+   alwayslink=1,
+)
+
+cc_binary(
+    name = "RenderSystem_GL3Plus.so",
```

Comment:
This name needs to be prepended with `lib` to be compatible with downstream ign-rendering dependency name